typo3-ter/cal/2010-03-02.yaml

```diff
@@ -0,0 +1,8 @@
+title:     "TYPO3-SA-2010-005: Blind SQL Injection vulnerability in extension Calendar Base (cal)"
+link:      http://typo3.org/teams/security/security-bulletins/typo3-extensions/typo3-sa-2010-005/
+cve:
```

Comment:
ah no sorry. The empty node is a valid representation of ``null`` in Yaml (in case it is possibl to use it). but including it would be safer though